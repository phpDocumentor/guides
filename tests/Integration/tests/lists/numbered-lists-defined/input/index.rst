==============
Document Title
==============

1.  First numbered list item.
2.  This is the second numbered list item.
#.  and then the third numbered list item.

Lorem Ipsum Dolor

3.  First numbered list item.
2.  This is the second numbered list item.
1.  and then the third numbered list item.

    1. This is an indented numbered list.
    #. This is the second item.

1. First
2. Second

Intermediary text or image.

3. Third