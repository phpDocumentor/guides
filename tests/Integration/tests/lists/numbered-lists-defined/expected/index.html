<!-- content start -->
    <div class="section" id="document-title">
            <h1>Document Title</h1>


<ol start="1" type="1">
    <li value="1">First numbered list item.</li>
    <li value="2">This is the second numbered list item.</li>
    <li>and then the third numbered list item.</li>
</ol>


    <p>Lorem Ipsum Dolor</p>



<ol start="3" type="1">
    <li value="3">First numbered list item.</li>
    <li value="2">This is the second numbered list item.</li>
    <li value="1">
    <p>and then the third numbered list item.</p>


<ol start="1" type="1">
    <li value="1">This is an indented numbered list.</li>
    <li>This is the second item.</li>
</ol>
</li>
</ol>

    </div>
<!-- content end -->
